autopush/endpoint.py

```diff
@@ -649,12 +649,11 @@ def post(self, router_type="", router_token="", uaid="", chid=""):
             return self._write_response(
```

Comment:
Sorry, think the highlighting was going wacky, it had this line flagged as green for code change, refresh changed it.